homeassistant/components/tuneblade_remote/coordinator.py

```diff
@@ -0,0 +1,49 @@
+"""Integration to integrate TuneBlade Remote devices with Home Assistant."""
+
+from datetime import timedelta
+import logging
+
+from pytuneblade import TuneBladeApiClient
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+SCAN_INTERVAL = timedelta(seconds=5)
+
+
+class TuneBladeDataUpdateCoordinator(DataUpdateCoordinator):
+    """Coordinator to fetch data from the TuneBlade hub."""
+
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        client: TuneBladeApiClient,
+        scan_interval: timedelta = SCAN_INTERVAL,
```

Comment:
why is the scan interval so low?

Reply:
No reason other than it made it snappy. Changed it to 10 but open to suggestions.

Reply:
I have spotify on 30 seconds. I think that's a nice compromise. As an improvement I also added a way to update the player state 1 second after a track ended. This way it updates quicker when needed.